mobile_app/packages/teigiii_api/doc/FrameStatsRequest.md

```
@@ -0,0 +1,17 @@
+# teigiii_api.model.FrameStatsRequest
+
+## Load the model package
+```dart
+import 'package:teigiii_api/api.dart';
+```
+
+## Properties
+Name | Type | Description | Notes
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)